content/en/docs/setup/keadm.md

```diff
@@ -15,20 +15,20 @@ Please refer [kubernetes-compatibility](https://github.com/kubeedge/kubeedge#kub
 
 ## Limitation
 
-- Currently support of `keadm` is available for Ubuntu and CentOS OS. RaspberryPi supports is in-progress.
+- Currently support of *keadm* is available for Ubuntu and CentOS OS. RaspberryPi supports is in-progress.
 - Need super user rights (or root rights) to run.
 
 ## Install keadm
 
-Run the command below to one-click install `keadm`.
+Run the command below to one-click install *keadm*.
```

Comment:
Same as above

Reply:
Okay will change that. 
Are we following the same documentation style that is followed by [kubernetes](https://kubernetes.io/docs/contribute/style/) project?

Reply:
@nilisha-jais Yes we are. Although it may not be followed everywhere at the moment.